cpmonitor/static/css/main.scss

```diff
@@ -74,3 +74,10 @@ figure.lz-chart {
     object-fit: contain;
   }
 }
+
+.lz-card-summary {
+  -webkit-line-clamp: 8;
```

Comment:
You use a lot of webkit-specific stuff. Will they work in other browsers?

Reply:
Does "a lot" refer to the three properties in the lz-card-summary class or is there more that I am currently not aware of?
If the lz-card-summary class is all that you are referring to, then yes. Generally I am always careful to check support. In this case the two properties and the value for display are supported by all major browsers:
https://developer.mozilla.org/en-US/docs/Web/CSS/-webkit-line-clamp
https://developer.mozilla.org/en-US/docs/Web/CSS/box-orient
(support is detailed at the bottom of the page, in case you are not familiar with the MDN docs)

Reply:
I noticed the stuff you deleted here, earlier, but didn't see you just moved it here. Sry. Okay. Still learning...